pkg/kubelet/dockershim/docker_stats_windows.go

```diff
@@ -52,6 +53,14 @@ func (ds *dockerService) getContainerStats(containerID string) (*runtimeapi.Cont
 
 	stats, err := hcsshimContainer.Statistics()
 	if err != nil {
+		if strings.Contains(err.Error(), "0x5") || strings.Contains(err.Error(), "0xc0370105") {
+			// When the container is just started quering for stats causes access errors
+			// This is transient; skip container for now
+			// These hcs errors do not have helpers exposed in public package so need to query for the known codes:
+			// https://github.com/microsoft/hcsshim/blob/master/internal/hcs/errors.go
+			klog.Errorf("Container is not in a state that stats can be accessed '%s': %v. This occurs when the container is created but not started.", containerID, err)
```

Comment:
Should we make this an info message since nothing is actually wrong and there is nothing actionable?

Reply:
I was following the same pattern as above https://github.com/kubernetes/kubernetes/blob/4a8b2d377ba51124c0277ed4e25afe2eeefda085/pkg/kubelet/dockershim/docker_stats_windows.go#L41-L43

If we do that here I think we should do it there too?

Reply:
Let's update both messages to log as Info at V(4).
That matches some similar logs messages i'm seeing in dockershim
https://github.com/kubernetes/kubernetes/blob/64a6ee4321d37740b84a5c5c7aaa67e0bf6b6508/pkg/kubelet/dockershim/docker_container.go#L76

@ehashman any concerns or comments with this?

Reply:
I know we're not yet on structured logging in this file yet but I don't think there's any issue with changing this to an Info, the current guidance is that expected errors are fine to log at the info level: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#using-errors

Reply:
I've updated to log using info